snapcraft.yaml

```diff
@@ -0,0 +1,18 @@
+name: bittorrent-tracker
+version: master
+summary: Simple, robust, BitTorrent tracker
+description: |
+  Node.js implementation of a BitTorrent tracker, client and server.
+
+apps:
+  bittorrent-tracker:
+    command: bittorrent-tracker
+    plugs: [network, network-bind]
+
+grade: devel
+confinement: strict
+
+parts:
```

Comment:
This "parts" section exports a re-usable part named "tracker", right? Should we name it "bittorrent-tracker" to be clearer?